### ISSUE TITLE
Can't Resolve "com.typesafe" % "sbt-mima-plugin" % "0.6.0"

### ISSUE THREAD

**Issue by mantisMolloy** [akka/akka-http#2785]
After clearing my `~/.ivy2/cache` I can no longer resolve "com.typesafe" % "sbt-mima-plugin" % "0.6.0".

This was working before clearing my ivy2 cache and I was able to build the project. I made no changes to my sbt config.

This issue has been reproduced by another github user (https://github.com/zwift-mark).

```
tom@earcandle:~/repos/test$ git clone https://github.com/akka/akka-http.git && cd akka-http && sbt
Cloning into 'akka-http'...
remote: Enumerating objects: 68103, done.
remote: Total 68103 (delta 0), reused 0 (delta 0), pack-reused 68103
Receiving objects: 100% (68103/68103), 11.74 MiB | 9.83 MiB/s, done.
Resolving deltas: 100% (33730/33730), done.
[info] Loading settings for project akka-http-build from plugins.sbt ...
[info] Loading project definition from /Users/tom/repos/test/akka-http/project
[info] Updating ProjectRef(uri("file:/Users/tom/repos/test/akka-http/project/"), "akka-http-build")...
[warn]     module not found: com.typesafe#sbt-mima-plugin;0.6.0
[warn] ==== typesafe-ivy-releases: tried
[warn]   https://repo.typesafe.com/typesafe/ivy-releases/com.typesafe/sbt-mima-plugin/scala_2.12/sbt_1.0/0.6.0/ivys/ivy.xml
[warn] ==== sbt-plugin-releases: tried
[warn]   https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/com.typesafe/sbt-mima-plugin/scala_2.12/sbt_1.0/0.6.0/ivys/ivy.xml
[warn] ==== local: tried
[warn]   /Users/tom/.ivy2/local/com.typesafe/sbt-mima-plugin/scala_2.12/sbt_1.0/0.6.0/ivys/ivy.xml
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/com/typesafe/sbt-mima-plugin_2.12_1.0/0.6.0/sbt-mima-plugin-0.6.0.pom
[warn] ==== local-preloaded-ivy: tried
[warn]   /Users/tom/.sbt/preloaded/com.typesafe/sbt-mima-plugin/0.6.0/ivys/ivy.xml
[warn] ==== local-preloaded: tried
[warn]   file:////Users/tom/.sbt/preloaded/com/typesafe/sbt-mima-plugin_2.12_1.0/0.6.0/sbt-mima-plugin-0.6.0.pom
[warn] ==== sbt-plugin-releases: tried
[warn]   https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/com.typesafe/sbt-mima-plugin/scala_2.12/sbt_1.0/0.6.0/ivys/ivy.xml
[warn] ==== typesafe-ivy-releases: tried
[warn]   https://repo.typesafe.com/typesafe/ivy-releases/com.typesafe/sbt-mima-plugin/scala_2.12/sbt_1.0/0.6.0/ivys/ivy.xml
[warn] ==== sonatype-releases: tried
[warn]   https://oss.sonatype.org/content/repositories/releases/com/typesafe/sbt-mima-plugin_2.12_1.0/0.6.0/sbt-mima-plugin-0.6.0.pom
[warn] ==== jcenter: tried
[warn]   https://jcenter.bintray.com/com/typesafe/sbt-mima-plugin_2.12_1.0/0.6.0/sbt-mima-plugin-0.6.0.pom
[warn]     ::::::::::::::::::::::::::::::::::::::::::::::
[warn]     ::          UNRESOLVED DEPENDENCIES         ::
[warn]     ::::::::::::::::::::::::::::::::::::::::::::::
[warn]     :: com.typesafe#sbt-mima-plugin;0.6.0: not found
[warn]     ::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn]     Note: Some unresolved dependencies have extra attributes.  Check that these dependencies exist with the requested attributes.
[warn]         com.typesafe:sbt-mima-plugin:0.6.0 (scalaVersion=2.12, sbtVersion=1.0)
[warn] 
[warn]     Note: Unresolved dependencies path:
[warn]         com.typesafe:sbt-mima-plugin:0.6.0 (scalaVersion=2.12, sbtVersion=1.0) (/Users/tom/repos/test/akka-http/project/plugins.sbt#L11-12)
[warn]           +- default:akka-http-build:0.1.0-SNAPSHOT (scalaVersion=2.12, sbtVersion=1.0)```

**Comment by jrudolph** [comment]
See https://github.com/lightbend/mima/issues/422. Preliminary fix in #2780

**Comment by jrudolph** [comment]
lightbend/mima#422 is fixed for now. Closing...
